pxemgr/ignition.go

```diff
@@ -7,7 +7,7 @@ import (
 	"io"
 	"reflect"
 
-	"github.com/coreos/ignition/config/types"
+	"github.com/coreos/ignition/config/v2_0/types"
```

Comment:
locking to ignition 2.0 as 2.1 is experimental/alpha and there are few breaking changes